src/scripts/settings/userSettings.js

```diff
@@ -253,6 +253,20 @@ export class UserSettings {
         return val !== 'false';
     }
 
+    /**
+     * Get or set 'Use Episode Images in Next Up and Continue Watching' state.
+     * @param {string|boolean|undefined} val - Flag to enable 'Use Episode Images in Next Up and Continue Watching' or undefined.
+     * @return {boolean} 'Use Episode Images in Next Up' state.
+     */
+    useEpisodeImagesInNextUpAndResume(val) {
+        if (val !== undefined) {
+            return this.set('useEpisodeImagesInNextUpAndResume', val.toString(), false);
+        }
+
+        val = this.get('useEpisodeImagesInNextUpAndResume', false);
+        return val === 'true';
+    }
```

Comment:
Send settings to the server. This way, the settings will be in-sync on different clients for the same user.
```suggestion
    useEpisodeImagesInNextUpAndResume(val) {
        if (val !== undefined) {
            return this.set('useEpisodeImagesInNextUpAndResume', val.toString(), true);
        }

        val = this.get('useEpisodeImagesInNextUpAndResume', true);
        return val === 'true';
    }
```